web/src/app/services/auth.service.ts

```diff
@@ -123,13 +127,15 @@ export class AuthService {
     };
 
     const normalisedUser = {
-      displayName: user.displayName,
+      twitter: {
+        displayName: user.displayName,
+        photoUrl: user.photoURL,
+        creationAt: user.metadata.creationTime,
+        lastSignInAt: user.metadata.lastSignInTime,
+        additionalUserInfo,
+        credentials,
```

Comment:
Ending `,` not consistent. I prefer using this, but it was removed in many places in a previous PR (won't block PR though, can fix in future PR)

Reply:
okay @eddiejaoude . Will change all through the code